SpatialGDK/Documentation/content/technical-overview/gdk-concepts.md

```diff
@@ -14,32 +14,32 @@ Key concepts in Unreal Engine map to concepts in the GDK, as shown in the table
+Offloading is one of the multiserver options for working with SpatialOS. It involves allocating the authority over specific groups of Actors to a server-worker instance other than your main Unreal server-worker instance. 
+
+For example, you could offload computationally expensive but latency-tolerant systems, such as advanced AI or large-scale background physics simulation, so that your main server-worker instance can run other systems at a larger scale.
 
 <%(Lightbox image="{{assetRoot}}assets/offloading-diagram.png")%>
-_Offloading: Offloaded Unreal server-worker instance has authority only over Red Actors and the Main Unreal server-worker instance that runs major game systems has authority over all Actors except the Red Actors._
+_Offloading: The offloaded server-worker instance has authority only over Red Actors, and the main Unreal server-worker instance that runs major game systems has authority over all other Actors._
 
 For more information, see [Offloading overview]({{urlRoot}}/content/workers/offloading-concept).
 
 ### Actor groups
-To facilitate offloading, we've created the concept of Actor groups to help you configure what Actor types that a given server-worker type has authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
+To facilitate offloading, we've created the concept of Actor groups to help you configure the Actor types that a given type of server-worker instance has authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.
 
-Before you start to use offloading in your game, ensure that you’re familiar with the [best practices for using offloading]({{urlRoot}}/content/workers/offloading-concept#best-practices) and [offloading workflow]({{urlRoot}}/content/workers/set-up-offloading).
+Before you start to use offloading in your game, make sure that you’re familiar with the [best practices]({{urlRoot}}/content/workers/offloading-concept#best-practices) and with how to [set up your game feature for offloading]({{urlRoot}}/content/workers/set-up-offloading).
 
 ### Zoning
-Because the GDK uses SpatialOS networking, you can have multiple server-worker instances simulating your game world. This allows you to extend the size of the world.
-
-We call this _zoning_ - splitting up the world into zones, known as “areas of authority”. Each area is simulated by one server-worker instance. This means that only one server-worker instance has authority to make updates to SpatialOS components at a time.
+Like [offloading](#offloading), zoning is a multiserver option for working with SpatialOS. It involves splitting up the world into zones, known as “areas of authority”, with a different server-worker instance responsible for each. A server-worker instance can make updates only to Actors that are in its area of authority.
```

Comment:
This could suggest that offloading and zoning are similar.
Also it's probs worth saying it's about sharing the server computation load ?
Also, we should probs never say "zones" and only ever "areas" because that's a synonym happening.

```suggestion
Zoning is another of the multiserver options when working with SpatialOS, it works differently to [offloading](#offloading). 
Zoning involves sharing the server computation load by splitting up the world into areas of authority, with a different server-worker instance responsible for the computation of each area. A server-worker instance can make updates only to Actors that are in its area of authority.
```